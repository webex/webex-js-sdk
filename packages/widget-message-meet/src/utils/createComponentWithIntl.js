import React from 'react';
import renderer from 'react-test-renderer';
import {IntlProvider} from 'react-intl';

function createComponentWithIntl(children, props = {locale: `en`}) {
  return renderer.create(
    <IntlProvider {...props}>
      {children}
    </IntlProvider>
  );
}

export default createComponentWithIntl;
