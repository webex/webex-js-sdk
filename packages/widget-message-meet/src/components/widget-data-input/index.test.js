import React from 'react';

import createComponentWithIntl from '../../utils/createComponentWithIntl';
import WidgetDataInput from '.';

describe(`WidgetDataInput component`, () => {

  const component = createComponentWithIntl(
    <WidgetDataInput />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
