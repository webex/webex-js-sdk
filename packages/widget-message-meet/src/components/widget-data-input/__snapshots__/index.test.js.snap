exports[`WidgetDataInput component renders properly 1`] = `
<div
  className="widget-container">
  <div
    className="spark-logo">
    <img
      role="presentation"
      src={Object {}} />
  </div>
  <form
    className="widget-props-form">
    <div
      className="field-wrapper">
      <input
        className="field-input"
        onChange={[Function]}
        placeholder="To User Email"
        type="text"
        value={undefined} />
    </div>
    <div
      className="field-wrapper">
      <input
        className="field-input"
        onChange={[Function]}
        placeholder="Your Access Token"
        type="text"
        value={undefined} />
    </div>
    <button
      className="props-submit"
      onClick={[Function]}>
      Chat
    </button>
    
  </form>
</div>
`;
